### PR TITLE
Added Community task driver for FreeBSD jails

### PR DESCRIPTION
**Pull request by cneira**
Added documentation for a FreeBSD task driver that leverages jails for container creation.

### CODE REVIEW

File: website/source/docs/drivers/external/jail-task-driver.html.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+---
+layout: "docs"
+page_title: "Drivers: jail-task-driver"
+sidebar_current: "docs-drivers-community-jail-task-driver"
+description: |-
+  The LXC task driver is used to run application containers using LXC.
```
Comment (preetapan) [inline website/source/docs/drivers/external/jail-task-driver.html.md]: This should say Jail task driver

Comment (cneira) [reply website/source/docs/drivers/external/jail-task-driver.html.md]: Thanks for reviewing!, I just fixed this reference.

**Comment by preetapan** [comment]
Thanks for your contribution @cneira. This isn't live on nomadproject.io yet, we are planning a release of Nomad 0.9.4 this week and this will be live on the site at that time.

**Comment by cneira** [comment]
Thank you very much @preetapan.

**Pull request merged**